tests/Api/Responses/admin/product_image/post_product_image_response.json

```diff
@@ -7,5 +7,5 @@
     "type": null,
     "path": "@string@.contains('sylius_original').endsWith('mugs.jpg')",
     "productVariants": [],
-    "position": 3
+    "position": 2
```

Comment:
it seems it was wrong earlier, as we can see [here](https://github.com/Sylius/Sylius/blob/2.0/tests/Api/DataFixtures/ORM/product/product_image.yaml#L1-L6)
the mug has to images, so the third one should have position 2.  It was because of lack of sorting [here](https://github.com/Sylius/Sylius/pull/16933/commits/8a34c92e80f411a23cceb726f2677ebface5232d#diff-fddc6cc6b10b559fa57b9f61a41c25335e423bd6491b47775c95210f6fcf6362)